app/components/EventItem/styles.css

```diff
@@ -48,6 +55,12 @@
   max-height: 80px;
   object-fit: contain;
   background: white;
+  margin: 5px 10px 5px 0;
```

Comment:
These should be spacing variables. It's important that all the spacings in the webapp are similar so that things don't feel off😁